### ISSUE TITLE
Sum of column from checked row

### ISSUE THREAD

**Issue by warrence** [ratiw/vuetable-2#287]
Hi, i have a column "amount" in vuetable, I would like to have a below vuetable to show the total mount selected, single check and also check all. Anyone know how to do that?

**Comment by ratiw** [comment]
Listen to `vuetable:checkbox-toggled` and `vuetable:checkbox-toggled-all` events and calculated the sum based on the selected item in the `selectedTo` array data.

**Comment by warrence** [comment]
Hi, but selectedTo array of data only consist of data on the current page?

Example I have 10 result per page. then I select 1st and 2nd row from page
1, then move to page2, in page 2 I selected 11th and 13th row, but in
selectedTo array data only have row 11th and 13th,  I wouldn't be able to
recalculate all the amount again when click on selectall

**Comment by ratiw** [comment]
No, the `selectedTo` will have all selected rows. That's why if you select rows from the first page and go to page 2 and come back to the first page again, you will see those selected rows are still marked as selected.

The real problem would be the data in the `tableData` which will always contain data from the current page. That will make it unlikely to sum values from other pages.

**Comment by warrence** [comment]
Oh yes that’s what I mean, any advise on how to recalculate Sum in this
case?

**Comment by ratiw** [comment]
You should use another local array (e.g. `selectedItems`) to add/remove the data record of the selected row to it by listening to `vuetable:checkbox-toggled`.

Then, you should be able sum the data from that local array instead, e.g. 

_Note_  
_The code is not complete and may not work correctly, but the idea is there._

```javascript
methods: {
  sumSelectedItems() {
    let total = 0
    return this.selectedItems.reduce( (total, item) => {
      return total + item.amount
    })
  }
}
```

**Comment by warrence** [comment]
Ok thanks for that.
